packages/node_modules/nav-frontend-tabs/src/tab.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import * as cn from 'classnames';
+import { omit } from 'nav-frontend-js-utils';
+
+const tabCls = (props) => cn('nav-frontend-tabs__tab-inner', {
+    'nav-frontend-tabs__tab-inner--aktiv': props.aktiv,
+    'nav-frontend-tabs__tab-inner--interaktiv': !props.aktiv,
+    'nav-frontend-tabs__tab-inner--disabled': props.disabled
+});
+
+export interface TabProps {
+    label?: React.ReactNode | React.ReactChild | React.ReactChildren;
+    aktiv: boolean;
+    onClick?: (event: React.MouseEvent<HTMLButtonElement>) => void;
+    disabled?: boolean;
+}
+
+class Tab extends React.Component<TabProps> {
+    getLabel() {
+        return (
+            <div className="nav-frontend-tabs__tab-label">
+                {this.props.label || this.props.children}
+            </div>
+        );
+    }
+
+    getInnerTab() {
+        if (!this.props.aktiv) {
+            return (
+                <button
+                    className={tabCls(this.props)}
+                    onClick={this.props.onClick}
+                    role="tab"
+                    aria-selected={false}
+                    disabled={this.props.disabled}
+                >
+                    {this.getLabel()}
+                </button>
+            );
+        } else {
+            return (
+                <div 
```

Comment:
Er det ikke litt funky å bytte ut `button` med `div` her?
Ser bare for meg at en skjermleser-bruker vil bruker knapp-hurtigtasten for å bla gjennom valgene, men da vil ikke den aktive tabben dukke opp i listen over tabs.

Reply:
Hm, litt usikker. Håper kanskje at `role="tablist"`, `role="tab"` og `aria-selected="true/false"` vil gjøre susen her, men ser at [noen](http://simplyaccessible.com/article/danger-aria-tabs/) faktisk anbefaler å ikke bruke noen roles overhodet.

Skal høre om David har anledning til å teste litt og gi oss noen anbefalinger.